include/fe/fe.h

```diff
@@ -58,13 +58,19 @@ struct FEOutputType
 template<>
 struct FEOutputType<LAGRANGE_VEC>
 {
-  typedef RealGradient type;
+  typedef RealVectorValue type;
```

Comment:
This is just for readability, right?  The old code was confusing, not actually broken in some subtle way?

Reply:
correct